File: reservations - SPA/reservations-spa/src/app/_models/Desk.ts
export class Desk{
    public name:string;
    public id:string;
    public officeId:string;
    public seats:number;

    constructor(name:string, id:string, officeId:string, seats:number) {
        this.name = name;
        this.id = id;
        this.officeId = officeId;
        this.seats = seats;
    }
}

File: reservations - API/reservations/Reservations.Core/Domain/Desk.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Reservations.Core.Domain
{
    public class Desk
    {

        public Desk()
        {
            UserDesks = new List<DeskReservation>();
        }
        public string Name { get;  set; }
        public Guid Id { get;  set; }
        public Guid OfficeId { get;  set; }
        public int Seats { get; set; }
        List<DeskReservation> UserDesks { get; set; }
    }
}
